_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -38,7 +38,7 @@ This rule applies to any element for which all the following is true:
 
 ## Expectation
 
-For each target element, all [text nodes][] in the [visible text content][] either match or are contained within the [accessible name][] of this target element, except for characters in the [text nodes][] used to express [non-text content][]. Leading and trailing [whitespace][] and difference in case sensitivity should be ignored.
+For each target element, all [text nodes][] in the [visible text content][] match in sequence and are contained within the [accessible name][] of this target element, except for characters in the [text nodes][] used to express [non-text content][]. Leading and trailing [whitespace][] and difference in case sensitivity should be ignored.
```

Comment:
I'm wondering if we should have a glossary definition for `match in sequence`. I know what you mean from being part of the conversations, but I don't think it reads very well. Some more explanation for how you are matching the sequence of characters exactly might be helpful.

Reply:
There is an existing definition for "matching characters" from [WAI Glossary Harmonization](https://docs.google.com/spreadsheets/d/16C3vsiNmrRBwHcNx_WhQYoMaCkUIc6YjWTfqfEnXU9c/edit#gid=0):

> A sequence of characters is considered to match another if, after removing leading and trailing space characters and replacing remaining occurrences of one or more space characters with a single space, the two sequences of characters are equal character-by-character, ignoring any differences in letter casing.

Would this work?
```suggestion
For each target element, all [text nodes][] in the [visible text content][] match characters and are contained within the [accessible name][] of this target element, except for characters in the [text nodes][] used to express [non-text content][]. Leading and trailing [whitespace][] and difference in case sensitivity should be ignored.
```